### DISCUSSION TITLE
The correct events are missing if the concurrent undo cycle end not come in the log

### DISCUSSION THREAD

**Discussion by logesh-encipherhealth** [microsoft/gctoolkit#299] [General]
The event will be published after the concurrent undo cycle come. If the concurrent undo cycle does not come, the event won't be published. So we can't get the correct parsed data what are the steps to get the correct output for that log? I will share one log for reference [https://drive.google.com/file/d/1ZvmgAa0vzXzOEb6LqFDDGvP9mYIDECyF/view?usp=sharing](url)

**Comment by kcpeppe** [comment]
The end of gc log event should flush the queues.. and if it doesn't, then this would be a bug. And it might be the case that it doesn't because missing a data point here or there isn't likely to make that much of a difference in the overall analysis but yeah, lets check that  to make sure it's doing the right thing

**Reply by logesh-encipherhealth** [reply]
Hi @kcpeppe  have you checked this logs Any leads?

**Comment by logesh-encipherhealth** [comment]
other tools parsing the above log
